.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: ruby
+rvm:
+  - 2.4
+env:
+  - DB_USERNAME='postgres' DB_PASSWORD=''
```

Comment:
make each env in its own line

Reply:
For each line, it will trigger one worker with env variable of that specific line, hence I believe this is the correct way to define env variable. 

You can refer to the docs https://docs.travis-ci.com/user/environment-variables/ at section `Defining Multiple Variables per Item #` for more information

Reply:
it's only true when you have more than 1 items in the matrix